src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -75,6 +82,7 @@ const getAuthenticatorData = function (authenticator, isVerifyAuthenticator) {
 
   case 'app':
     Object.assign(authenticatorData, {
+      label: getOVLabel(authenticator.value.methodType),
```

Comment:
we need to check for isVerifyAuthenticator here?

Reply:
yeah